src/core/nsfs.js

```diff
@@ -104,7 +104,11 @@ async function main(argv = minimist(process.argv.slice(2))) {
         object_sdk._get_bucket_namespace = async bucket_name => {
             const existing_ns = namespaces[bucket_name];
             if (existing_ns) return existing_ns;
-            const ns_fs = new NamespaceFS({ bucket_path: fs_root + '/' + bucket_name, bucket_id: '000000000000000000000000' });
+            const ns_fs = new NamespaceFS({
+                bucket_path: fs_root + '/' + bucket_name,
+                bucket_id: '000000000000000000000000',
+                namespace_resource_id: '9999'
```

Comment:
can we pass undefined `namespace_resource_id` and then namespace_fs will avoid sending reports?
because in standalone more there is nowhere to send it.